sdk/core/core/src/az_span_builder.c

```diff
@@ -23,6 +25,59 @@ AZ_NODISCARD az_result az_span_builder_append_byte(az_span_builder * const self,
   return AZ_OK;
 }
 
+AZ_INLINE uint8_t _az_decimal_to_ascii(uint8_t const d) {
```

Comment:
I don't think we need this; I think you can pass a uint8_t to a function that wants a long long